textproc/opensp/Portfile

```diff
@@ -1,40 +1,46 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
+PortSystem      1.0
 
-name		opensp
-version		1.5.2
-revision	2
-categories	textproc
-maintainers	nomaintainer
-description	Collection of SGML/XML tools
-homepage	http://openjade.sourceforge.net/
-platforms	darwin
-license     MIT
+name            opensp
+version         1.5.2
+revision        2
+categories      textproc
+maintainers     nomaintainer
+description     Collection of SGML/XML tools
+homepage        https://openjade.sourceforge.net
+license         MIT
 
-long_description OpenSP is a collection of SGML/XML tools that has \
-		 forked from James Clark's SP suite.  These tools area \
-		 used to parse, validate, and normalize SGML and XML \
-		 files.
+long_description \
+                OpenSP is a collection of SGML/XML tools that has \
+                forked from James Clark's SP suite. These tools area \
+                used to parse, validate and normalize SGML and XML files.
 
-master_sites	sourceforge:project/openjade/opensp/${version}
-distname	OpenSP-${version}
+master_sites    sourceforge:project/openjade/opensp/${version}
+distname        OpenSP-${version}
 
-checksums	md5 670b223c5d12cee40c9137be86b6c39b \
-		sha1 b4e903e980f8a8b3887396a24e067bef126e97d5
+checksums       rmd160  6e671a267562c5b9283e7e6504e8339cb1d86850 \
+                sha256  57f4898498a368918b0d49c826aa434bb5b703d2c3b169beb348016ab25617ce \
+                size    1521390
 
 patchfiles      patch-automake-1.13.diff
 
 use_autoreconf  yes
 autoreconf.args -fvi
 
-depends_lib	port:gettext
+depends_lib     port:gettext
 
-configure.args	--enable-http --disable-dependency-tracking \
-		--enable-default-catalog=${prefix}/share/sgml/catalog \
-		--mandir=${prefix}/share/man --disable-doc-build
+configure.args  --enable-http --disable-dependency-tracking \
+                --enable-default-catalog=${prefix}/share/sgml/catalog \
+                --mandir=${prefix}/share/man --disable-doc-build
 
-configure.env	"LIBS=-L${prefix}/lib -lintl"
-configure.cxxflags-append   -std=c++03
+configure.env   "LIBS=-L${prefix}/lib -lintl"
+
+# https://trac.macports.org/ticket/67301
+if {[string match *gcc-4.* ${configure.compiler}]} {
+    configure.cxxflags-append   -std=c++98
+} else {
+    configure.cxxflags-append   -std=c++03
+}
```

Comment:
I would probably just change it unconditionally (unless you've tried that under clang and it causes the build to fail).

Let's change the ticket reference to #64609.

```suggestion

# https://trac.macports.org/ticket/64609
configure.cxxflags-append   -std=c++98
```